ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapperTest.java

```diff
@@ -132,7 +138,7 @@ public void mapTheirDetailsFromCCDClaimWithNoResponse() {
         Claim.ClaimBuilder claimBuilder = Claim.builder();
 
         //when
-        mapper.from(claimBuilder, ccdDefendant);
+        mapper.from(claimBuilder, CCDCollectionElement.<CCDDefendant>builder().value(ccdDefendant).build());
```

Comment:
we can make use of MapperUtil to build a common method to build ccdcollectionelement with value and return. That will be useful in all test cases